src/jvm/test/resources/expected_logs/switch_as_first_method_event.txt

```diff
@@ -6,43 +6,43 @@
 | ----------------------------------------- |
 
 The following interleaving leads to the error:
-| ------------------------------------------------------------------------------------------------------------------ |
-|      Thread 1       |                                           Thread 2                                           |
-| ------------------------------------------------------------------------------------------------------------------ |
-|                     | incTwiceAndGet(): 4                                                                          |
-|                     |   incAndGet(): 1                                                                             |
-|                     |   incAndGet(): 4                                                                             |
-|                     |     incAndGetImpl(): 4                                                                       |
-|                     |       SwitchAsFirstMethodEventTest.getCounter$volatile$FU(): AtomicIntegerFieldUpdaterImpl#1 |
-|                     |       switch                                                                                 |
-| incTwiceAndGet(): 3 |                                                                                              |
-|                     |       counter.incrementAndGet(): 4                                                           |
-|                     |   result: 4                                                                                  |
-| ------------------------------------------------------------------------------------------------------------------ |
+| ------------------------------------------------------------------------------------- |
+|      Thread 1       |                            Thread 2                             |
+| ------------------------------------------------------------------------------------- |
+|                     | incTwiceAndGet(): 4                                             |
+|                     |   incAndGet(): 1                                                |
+|                     |   incAndGet(): 4                                                |
+|                     |     incAndGetImpl(): 4                                          |
+|                     |       getCounter$volatile$FU(): AtomicIntegerFieldUpdaterImpl#1 |
```

Comment:
@dmitrii-artuhov, please take a look at this line -- it seems redundant

Reply:
I think, it is not: the method is generated by the atomicfu library, if I remove the `$volatile$fu` suffix, I will get the name `[get/set]FieldName`, here it will be `getCounter` which is completely unexpected for the user, since he did not declare anything named like this. Or in case he did, we would show the name of the method which wasn't called in fact, because method from atomicfu was intended